docs/source/api/arduino.rst

```diff
@@ -0,0 +1,134 @@
+Arduino API
+===========
+
+The `Arduino <https://store.arduino.cc/arduino-uno-rev3>`__
+provides a total of 18 pins for either digital input or output (labelled
+2 to 13 and A0 to A5), including 6 for analogue input (labelled A0 to
+A5).
+
+Accessing the Arduino
+---------------------
+
+The Arduino can be accessed using the ``arduino`` property of
+the ``Robot`` object.
+
+.. code:: python
+
+   my_arduino = r.arduino
+
+You can use the GPIO *(General Purpose Input/Output)* pins for anything,
+from microswitches to LEDs. GPIO is only available on pins 2 to 12, and A0 to A5
```

Comment:
Removing the comma because the sentence flows weirdly when it's left there. Including pin 13 for consistency with the rest of the text.

```suggestion
from microswitches to LEDs. GPIO is only available on pins 2 to 13 and A0 to A5
```